tests/test_cases/test_iteration_mixedlang/test_iteration.py

```diff
@@ -75,3 +76,13 @@ async def recursive_discovery_boundary(dut):
     actual = recursive_dump(dut.i_vhdl, tlog)
     tlog.info("Found a total of %d things", actual)
     assert actual == expected
+
+
+@cocotb.test
+async def discovery_method(dut):
+    """Verify that the different discovery methods work."""
+
+    # this is a verilog toplevel, so we should not be finding a VHDL object
```

Comment:
```suggestion
    # this is a Verilog toplevel, so we should not be finding a VHDL object
```
Very nitpicky, I admit :)